### PR TITLE
Samples/gulp

### PR DESCRIPTION
**Pull request by targunp**
Added basic gulp + express sample

### CODE REVIEW

File: appengine/gulp/Gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+var gulp = require('gulp');
+var jshint = require('gulp-jshint');
+var cssnano = require('gulp-cssnano');
+
+// Lint Task
+gulp.task('lint', function() {
+    return gulp.src(['Gulpfile.js', 'src/**/*.js'])
```
Comment (jmdobry) [inline appengine/gulp/Gulpfile.js]: Can you reformat this file with 2-space indentation instead of 4? Thanks.

File: appengine/gulp/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  "dependencies": {
+    "body-parser": "^1.14.2",
+    "cookie-parser": "^1.4.1",
+    "debug": "^2.2.0",
+    "express": "^4.13.4",
+    "jade": "^1.11.0",
+    "morgan": "^1.6.1",
+    "serve-favicon": "^2.3.0"
+  },
+  "devDependencies": {
```
Comment (jmdobry) [inline appengine/gulp/package.json]: Can you move these into `dependencies`? Right now `devDependencies` won't get installed on deploy.

**Comment by jmdobry** [comment]
Just a couple fixes needed. After they're fixed, can you squash into 1 commit? Then I'll merge.

**Comment by targunp** [comment]
Sounds good, thanks

**Comment by jmdobry** [comment]
Feel free to re-open if you're able to fix the conflicts or give me edit access (you should see an "Allow Maintainers to Edit" checkbox on this PR.

**Pull request closed without merging**